src/hooks.py

```diff
@@ -0,0 +1,27 @@
+import arc
+import hikari
+import typing
+
+
+def restrict_to_roles(
```

Comment:
I will push a fix for these hooks.

Reply:
Hooks are now functional, but they need a bit of work to make them easier to use.